SerialPrograms/Source/CommonFramework/OCR/OCR_Routines.cpp

```diff
@@ -97,6 +97,14 @@ const std::vector<TextColorRange>& BLACK_OR_WHITE_TEXT_FILTERS(){
     };
     return filters;
 }
+const std::vector<TextColorRange>& BLUE_TEXT_FILTERS() {
+    static std::vector<TextColorRange> filters{
+        {0xFF3287A2, 0xFFA7F4FF},
+        {0xFF2283A2, 0xFF8EF1FF},
+        {0xFF428BA2, 0xFFC1F7FF},

```

Comment:
Were these found empirically? Or copied from somewhere?

Reply:
Took a screenshot in-program and found the color value used, then went 20% darker, 20% lighter. The two other lines are more/less saturation.